server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -100,6 +105,16 @@ public void upgradeMetadata(ServerContext context) {
     removeCompactColumnsFromTable(context, AccumuloTable.METADATA.tableName());
   }
 
+  private void createFateTable(ServerContext context) {
```

Comment:
Should this allow for the table to exist?  If the upgrade were to get past this step, but fail later on.  On a  rerun of the upgrade, then this may fail if the table was created?

I think we need to allow the upgrade to be attempted multiple times, but I don't think we require that the upgrade undo / rewind steps that it may have taken.

This is assuming that the presence of the fate table would not impact running the earlier versions - it would just be ignored and FATEs would be processed with the original code until the upgrade was successful.

Reply:
>Should this allow for the table to exist? If the upgrade were to get past this step, but fail later on. On a rerun of the upgrade, then this may fail if the table was created?

I think it does, right? I set the exists policy to `SKIP`. Did I miss something else?

Reply:
Actually that looks okay then.